src/Extension/FluentFilteredExtension.php

```diff
@@ -34,6 +34,14 @@ class FluentFilteredExtension extends DataExtension
      */
     const SUFFIX = 'FilteredLocales';
 
+    /**
+     * Allow the filtered locale behaviour to be skipped for draft stage only
+     *
+     * @var bool
+     * @config
+     */
+    private static bool $apply_filtered_locales_to_stage = false;
```

Comment:
This is set to `true` in `_config/model.yml` - is it correct to change it here? If so, why?

Reply:
It's either a mistake or an upstream change. Either way it needs to match the `model.yml`. Will make the change.